canada_post/service/contract_shipping.py

```diff
@@ -476,7 +480,7 @@ class GetGroups(ServiceBase):
     URL = 'https://{server}/rs/{customer}/{mobo}/group'
     log = logging.getLogger('canada_post.service.contract_shipping'
                             '.GetGroups')
-    headers = {'Accept': 'application/vnd.cpc.manifest-v4+xml',
+    headers = {'Accept': 'application/vnd.cpc.shipment-v7+xml',
```

Comment:
I changed it to match the Canada Post documentation:
https://www.canadapost.ca/cpo/mc/business/productsservices/developers/services/shippingmanifest/groups.jsf
Accept: application/vnd.cpc.shipment-v7+xml (Note: */* in place of the header value will return an error)